internal/transformer/transform.go

```diff
@@ -61,7 +61,7 @@ func CommandValuesToEventDTO(cvs []*models.CommandValue, deviceName string, sour
 		}
 
 		// perform data transformation
-		if config.Device.DataTransform {
+		if dataTransform {
```

Comment:
modify line 50 from
`readings := make([]dtos.BaseReading, 0, config.Device.MaxCmdOps)`
to 
`readings := make([]dtos.BaseReading, 0, len(cvs))`

then, we can remove the `config` from this function for good.  I forget the reason we used `config.Device.MaxCmdOps` in the first place.  It seems a wrong usage.